sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2005,71 +1967,63 @@ object functions {
   def signum(columnName: String): Column = signum(Column(columnName))
 
   /**
-   * @param e angle in radians
-   * @return sine of the angle, as if computed by [[java.lang.Math#sin]]
+   * Computes the sine of the given value. Units in radians.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def sin(e: Column): Column = withExpr { Sin(e.expr) }
 
   /**
-   * @param columnName angle in radians
-   * @return sine of the angle, as if computed by [[java.lang.Math#sin]]
+   * Computes the sine of the given column.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def sin(columnName: String): Column = sin(Column(columnName))
 
   /**
-   * @param e hyperbolic angle
-   * @return hyperbolic sine of the given value, as if computed by [[java.lang.Math#sinh]]
+   * Computes the hyperbolic sine of the given value.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def sinh(e: Column): Column = withExpr { Sinh(e.expr) }
 
   /**
-   * @param columnName hyperbolic angle
-   * @return hyperbolic sine of the given value, as if computed by [[java.lang.Math#sinh]]
+   * Computes the hyperbolic sine of the given column.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def sinh(columnName: String): Column = sinh(Column(columnName))
 
   /**
-   * @param e angle in radians
-   * @return tangent of the given value, as if computed by [[java.lang.Math#tan]]
+   * Computes the tangent of the given value. Units in radians.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def tan(e: Column): Column = withExpr { Tan(e.expr) }
 
   /**
-   * @param columnName angle in radians
-   * @return tangent of the given value, as if computed by [[java.lang.Math#tan]]
+   * Computes the tangent of the given column.
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def tan(columnName: String): Column = tan(Column(columnName))
 
   /**
-   * @param e hyperbolic angle
-   * @return hyperbolic tangent of the given value, as if computed by [[java.lang.Math#tanh]]
```

Comment:
Oh, but the `@param` and `@return` were fine ..  I thought a doc like:

```
  * Computes the hyperbolic tangent of the given value.  

  * @param e hyperbolic angle
  * @return hyperbolic tangent of the given value, as if computed by [[java.lang.Math#tanh]]
```

Reply:
This way the description and the return value text would be redundant. For this reason it was proposed [during the discussion of the issue](https://issues.apache.org/jira/browse/SPARK-23329?focusedCommentId=16354178&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16354178) to keep only the `@return` part.